src/core/inline-idl-parser.js

```diff
@@ -4,13 +4,13 @@
 import hyperHTML from "hyperhtml";
 import { showInlineError } from "./utils";
 
+const exceptionRegex = /\B"([^"]*)"\B/; // {{ "SomeException" }}
 const methodRegex = /(\w+)\((.*)\)$/;
 const slotRegex = /^\[\[(\w+)\]\]$/;
 // matches: `value` or `[[value]]`
 // NOTE: [[value]] is actually a slot, but database has this as type="attribute"
 const attributeRegex = /^((?:\[\[)?(?:\w+)(?:\]\])?)$/;
-const enumRegex = /^(\w+)\["([\w ]+)"\]$/;
-const enumValueRegex = /\B"([^"]*)"\B/;
+const enumRegex = /^(\w+)\["([\w- ]+)"\]$/;
```

Comment:
@sidvishnoi Why are some wrapped by `\B` and others not?

Reply:
Interesting. 
Not sure, must have been done to "make things work" at that time.  I think we can remove `\B`